ui/scss/user-dashboard/_admin.scss

```diff
@@ -242,25 +242,39 @@ table {
   }
 }
 
-.user-management-dialog .popover {
-  max-width: 500px;
```

Comment:
Without this rule, the popover size forced wrapping too early

Reply:
Doesn’t look like it is anywhere else in the new set of styles

Reply:
Done